### ISSUE TITLE
[material]  Custom View.AccessibilityDelegate does not work as expected on MaterialAutoCompleteTextView

### ISSUE THREAD

**Issue by qffan** [material-components/material-components-android#1886]
Artifact used: com.google.android.material:material:1.2.0
Version used:
Theme used:
Devices/Android versions reproduced on: Pixel 4, Android 11; Samsung A 50, Android 10

It is very possibly our own error but we did not find any helpful information on other platforms.
So we decide to try it here.

Source code:

<com.google.android.material.textfield.TextInputLayout
    ...
    style="@style/Widget.MaterialComponents.TextInputLayout.*.ExposedDropdownMenu">

    <AutoCompleteTextView
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
                 android:completionThreshold="0"
        android:inputType="none"
    />

</com.google.android.material.textfield.TextInputLayout>

The Problem:
It should be a not editable dropdown list. By default, TalkBack reads "Double-tap to edit text, double tab und hold to long press, edit options available... " on it.
Voice Assistant of Samsung reads "Double tab to enter text, double-tap and hold to long-press"

We want to change it to something like "double-tap to select XXX from a list, ...."

So we wrote a custom AccessibilityDelegate:

class DropDownListAccessibilityDelegate(view: TextInputLayout) : View.AccessibilityDelegate() {
    override fun onInitializeAccessibilityNodeInfo(
        host: View,
        info: AccessibilityNodeInfo
    ) {
        super.onInitializeAccessibilityNodeInfo(host, info)
        // A custom action description.
        val description =
            host.resources.getText(R.string.custom_dropdown_list_click_action)
        val customClick = AccessibilityNodeInfo.AccessibilityAction(
            AccessibilityNodeInfoCompat.ACTION_CLICK,
            description
        )
        info.addAction(customClick)
    }
}

If We set the custom delegate to TextInputLayout, nothing changes and the dropdown list shows after double click
If We set it to AutoCompleteTextView, the talkback text does not change, but the dropdown list does not show after double click.

We are not sure if it is a problem with the framework or we should change our delegate.
We used a similar AccessibilityDelegate on buttons, and it works.

**Comment by wcshi** [comment]
Please see the "Exposed Dropdown menu demos" in the catalog app, there are two examples of non-editable exposed dropdown menu that should have Talkback behavior you desire.

**Comment by qffan** [comment]
@wcshi Thank you for the answer. Actually, our XML codes are right.  I think the most import part is these two lines:

**style="@style/Widget.MaterialComponents.TextInputLayout.*.ExposedDropdownMenu" 
android:inputType="none"**

But anyhow we have also this line of code:
**binding.dropdownmenu.inputType = 0**
which changes the Talkback behavior back to EditText!  

After removing this line, it works perfectly.

**Comment by mmaya15** [comment]
Hey @qffan 

I am working on same changes that you did in above it's not working for me. can you please help me to fix it?
